openapi.md

```diff
@@ -0,0 +1,74 @@
+# Generating the OpenApi Spec (*.yaml)
```

Comment:
There are several lines with whitespaces at the end and also some unnecessary blank lines. Maybe you can reformat this file?